client/src/slate-helpers/slate-change-mutations/index.js

```diff
@@ -0,0 +1,17 @@
+import { adjustCursorIfAtEdge } from "./adjustCursorIfAtEdge";
```

Comment:
Does `export { ... } from ...` work?

Reply:
Yeah I think that makes more sense. I'll go ahead and make the change.